orchagent/neighorch.cpp

```diff
@@ -137,16 +137,24 @@ void NeighOrch::doTask(Consumer &consumer)
         }
 
         string alias = key.substr(0, found);
-        Port p;
 
-        if (!gPortsOrch->getPort(alias, p))
+        if (alias == "eth0" || alias == "lo" || alias == "docker0")
         {
             it = consumer.m_toSync.erase(it);
             continue;
         }
 
+        Port p;
+        if (!gPortsOrch->getPort(alias, p))
+        {
+            SWSS_LOG_INFO("Port %s doesn't exist", alias.c_str());
+            it++;
+            continue;
```

Comment:
but what under what scenario will the port name gets wrong? and if neighsyncd gets restarted, it will lose its previous memory, which will also cause trouble. if neighbor updates are sent periodically, what about other netlink messages?